### DISCUSSION TITLE
How do I update the version of wine being used?

### DISCUSSION THREAD

**Discussion by mistskyy** [The-Wineskin-Project/WineskinServer#392] [Q&A] [answered]
When using winetricks I get the following: your version of wine 6.xx isn't supported upstream. Upgrade to 7.X.

I tried installing the latest version of wine via homebrew but it did nothing.

Is it even possible?

**Comment by Gcenx** [comment]
If that’s an already created wrapper you’ll need to use the update button within the winetricks menu as I’d rebased against upstream winetricks two days ago and tweaked a couple verbs today.

**Comment by Gcenx** [comment, marked as answer]
Wait no I miss read…

You’d need to use a WineCX22 Engine as those are based on wine-7.7
